src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+server.port=9090
```

Comment:
Would it potentially be better to use a different port to timecard here in case we want to run both individually locally for any debugging/test reason..

Reply:
The most typical use case I foresee is debugging one single service and running the rest of services as callisto-localdev containers, which are on completely different port range (50XXX). This would only be a problem if we needed to debug 2 timecard and person services at the same time and even then we can change server.port temporarily.
Happy to change the port value if you still think there is value